docs/architecture/README.md

```diff
@@ -0,0 +1,32 @@
+# Service Fabric Core Subsystem Architecture
+
+Service Fabric is composed of several subsystems working together to achieve the promise of the platform: to remove the complexities of distributed computing from developers who want to build highly available, resilient, and massively scalable cloud servcices across public and private clouds, and operating systems. 
+
+As you can imagine, it takes a lot of code to materialize Service Fabric. To make it a bit easier for you to navigate through our sources, you can explore our codebase by clicking on a subsystem in the architectural graphic below. As of today, this includes the core subsystems that make up the Service Fabric Standalone product. This will grow over time as we move to open development in this repo, which will include the full product that runs today in Azure, with more components across platform services (e.g., our Chaos Experimentation system and API), build tools, and tests. 
```

Comment:
I think this tone doesn't really sit right.  This sounds more like a blog post than a time invariant living document. A few examples:
"As of today" -- This README isn't dated and I don't think using time in it makes sense
"our sources" -- this seems at odds with being open source.  We don't want a tone that this is "our source code"... it's open source code to be maintained by a community.  We don't own the code anymore

I think this segment should be briefer and focus on how to use the chart and the overall architecture